ee-feature-pack/galleon-shared/src/main/resources/modules/system/layers/base/org/apache/activemq/artemis/journal/main/module.xml

```diff
@@ -39,6 +39,7 @@
         <module name="java.desktop"/>
         <module name="java.logging"/>
         <module name="java.management"/>
+        <module name="java.xml"/>
```

Comment:
This is not required

Reply:
Yes, you're correct @ehsavoie This change in journal is not needed in upstream. Thanks.